packages/clay-popover/stories/index.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayPopover, {ClayTooltipListener} from '../src';
+import React from 'react';
+import {select} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/react';
+
+const positions = [
+	'top',
+	'top-left',
+	'top-right',
+	'bottom',
+	'bottom-left',
+	'bottom-right',
+	'left',
+	'left-top',
+	'left-bottom',
+	'right',
+	'right-top',
+	'right-bottom',
+] as const;
+
+storiesOf('ClayPopover', module)
+	.add('popover', () => (
+		<ClayPopover
+			alignPosition={select('Align Position', positions, 'bottom')}
+		>
+			<ClayPopover.Header>{'Popover'}</ClayPopover.Header>
+			<ClayPopover.Body>
+				{`Viennese flavour cup eu, percolator froth ristretto mazagran
+				caffeine. White roast seasonal, mocha trifecta, dripper caffeine
+				spoon acerbic to go macchiato strong.`}
+			</ClayPopover.Body>
+		</ClayPopover>
+	))
+	.add('tooltip', () => (
+		<ClayPopover
+			alignPosition={select('Align Position', positions, 'bottom')}
+			show
+			tooltip
+		>
+			<ClayPopover.Body>{'Tooltip'}</ClayPopover.Body>
+		</ClayPopover>
+	))
+	.add('w/ app context', () => (
+		<div style={{padding: 50}}>
+			<ClayTooltipListener autoAlign={false} />
```

Comment:
> Ideally we would only want to wrap our app once, and then infer the tooltip from the title attribute right?

Yeah.

> The other option is to instantiate the hook for every instance of a tooltip we have, this seems overly complicated for users though.

I think this would not be ideal here, we may have a lot of tooltips scattered around the application and we can generate a bloat for the application.

Reply:
Just to clarify, if we go with option 1, we would still need to use some sort of delegate type event correct?

Reply:
Yeah, I added a few comments below talking about it, which might be a better solution.